roles/StackStorm.st2smoketests/tasks/main.yml

```diff
@@ -50,6 +50,14 @@
   tags:
     - smoke-tests
 
+- name: Verify st2 pack is installed
+  environment:
+    ST2_AUTH_TOKEN: "{{ st2_token.stdout }}"
+  command: "st2 pack install st2"
```

Comment:
`command` content looks like copy-pasted by mistake from the previous task.

Reply:
Also we'll need to remove previous task as pack install now happens in `st2` role